ceph/iscsi/OWNERS

```diff
@@ -12,6 +12,9 @@ reviewers:
   - rahullepakshi
   - jprakash-redhat
   - krishnaramaswamy
+  - chaitanyadommeti
+  - sunilkumarn417
```

Comment:
Please don't add `sunilkumarn417` as he is one of the high level approvers. He shouldn't be part of reviewers group.